source/usb/usb_lib.c

```diff
@@ -2281,11 +2438,87 @@ const struct {
 #if (USBD_HID_ENABLE)
     USBD_STR_VAL(HID_STRDESC),
 #endif
+#if (USBD_WEBUSB_ENABLE)
+    USBD_STR_VAL(WEBUSB_STRDESC),
+#endif
 #if (USBD_MSC_ENABLE)
     USBD_STR_VAL(MSC_STRDESC),
 #endif
 };
 
+#if (USBD_WEBUSB_ENABLE)
```

Comment:
Trying to test this I was building the micro:bit DAPLink bootloader (kl26z_bl) on this branch, and in my environment this `if` macro ends up evaluating to `true`, so the `else` in line 2510 isn't compiled and the `USBD_EndPoint0_Setup_WebUSB_ReqToDevice ` symbol is never resolved:
```
*** Using Compiler 'V5.06 update 5 (build 528)', folder: 'C:\Keil_v5\ARM\ARMCC\Bin'
Build target 'kl26z_bl'
Before Build - User command #1: ..\..\..\tools\pre_build_script.bat
(env) C:\workspace\microbit-foundation\daplink-websub\projectfiles\uvision\kl26z_bl>python ..\..\..\tools\pre_build_script.py 
#> Pre-build script start
#> Getting git SHA
#> Checking for local changes
#> Keeping git version file since it didn't need to change
#> Pre-build script completed
compiling usb_config.c...
linking...
.\build\kl26z_bl.axf: Error: L6218E: Undefined symbol USBD_EndPoint0_Setup_WebUSB_ReqToDevice (referred from usbd_core.o).
Not enough information to list image symbols.
Not enough information to list load addresses in the image map.
Finished: 2 information, 0 warning and 1 error messages.
".\build\kl26z_bl.axf" - 1 Error(s), 0 Warning(s).
Target not created.
```
Can you replicate when building the kl26z_bl? Is it an issue on my configuration, or is it a known issue?

Reply:
@microbit-carlos Thanks for pointing it out. I've just fixed it for bootloaders.